ml-pipeline/content.md

```diff
@@ -22,10 +22,20 @@ Run ```pip install mlflow[extras]==1.1.0 mflux-ai kedro==0.15.0 keras==2.2.4 ten
 In this tutorial we will create a complete machine learning pipeline using [Kedro](https://github.com/quantumblacklabs/kedro).
 We will create a pipline for the video classification problem.
 
+## Why Kedro?
+
+The main advantage of Kedro compared to competitors like Azure and AWS is that it is easy and fast to go from an experimentation stage to production.
```

Comment:
Instead of refering to cloud providers (Azure), refer to _specific_ cloud services (e.g. Azure Data Factory) that are alternatives to Kedro